### DISCUSSION TITLE
Curso: Fullstack .Net -  Padronizando as Respostas

### DISCUSSION THREAD

**Discussion by whoiswentz** [balta-io/forum#651] [Tira Dúvidas] [answered]
Gostaria de entender melhor o porque de criar uma classe de Response, não seria melhor utilizar o TypedResults? ou há alguma diferença muito grande entre essas abordagens

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @whoiswentz, como vai?

O TypedResult aceita como resultado um `object` que pode ser qualquer coisa. Logo o resultado da resposta seria imprevisível, hora poderia ser uma lista, hora poderia ser um objeto.

Utilizamos o `Response` para padronizar o retorno dos endpoints. Tem mais detalhes sobre isto neste curso:
https://balta.io/cursos/fundamentos-aspnet

IMPORTANTE:
Recomendo fazer todos os cursos da carreira .NET - https://balta.io/carreiras
